bitnami/discourse/README.md

```diff
@@ -48,207 +48,193 @@ The command removes all the Kubernetes components associated with the chart and
+| `sidekiq.readinessProbe.failureThreshold`    | Minimum consecutive failures for the probe                        | `6`    |
+| `sidekiq.readinessProbe.successThreshold`    | Minimum consecutive successes for the probe                       | `1`    |
+| `sidekiq.customLivenessProbe`                | Custom liveness probe to execute (when the main one is disabled)  | `{}`   |
+| `sidekiq.customReadinessProbe`               | Custom readiness probe to execute (when the main one is disabled) | `{}`   |
+| `sidekiq.extraEnvVars`                       | An array to add extra env vars                                    | `[]`   |
+| `sidekiq.extraEnvVarsCM`                     | Array to add extra configmaps                                     | `[]`   |
+| `sidekiq.extraVolumeMounts`                  | Additional volume mounts                                          | `[]`   |
+
 
 ### Volume Permissions parameters
 
-| Parameter                             | Description                                                                                                                                               | Default                                                 |
-|---------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------|
-| `volumePermissions.enabled`           | Enable init container that changes volume permissions in the data directory (for cases where the default k8s `runAsUser` and `fsUser` values do not work) | `false`                                                 |
-| `volumePermissions.image.registry`    | Init container volume-permissions image registry                                                                                                          | `docker.io`                                             |
-| `volumePermissions.image.repository`  | Init container volume-permissions image name                                                                                                              | `bitnami/bitnami-shell`                                 |
-| `volumePermissions.image.tag`         | Init container volume-permissions image tag                                                                                                               | `"10"`                                                  |
-| `volumePermissions.image.pullSecrets` | Specify docker-registry secret names as an array                                                                                                          | `[]` (does not add image pull secrets to deployed pods) |
-| `volumePermissions.image.pullPolicy`  | Init container volume-permissions image pull policy                                                                                                       | `Always`                                                |
```

Comment:
Did these parameters disappeared?

Reply:
They aren't in the original `values.yaml` https://github.com/bitnami/charts/blob/0e8a4f1135c4b39cf866b309312cf822ae4c4bc5/bitnami/discourse/values.yaml#L284